publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/exception/ChannelClosureException.java

```diff
@@ -0,0 +1,24 @@
+/*
+    Copyright 2022 Ericsson AB.
+    For a full list of individual contributors, please see the commit history.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+package com.ericsson.eiffel.remrem.publish.exception;
+
+public class ChannelClosureException  extends Exception{
```

Comment:
I'd suggest to extend from IOException, as it's an I/O-related exception. In addition the exception don't need to be explicitly listed in method exception list.